src/test/java/teammates/test/driver/HtmlHelper.java

```diff
@@ -385,6 +389,8 @@ private static String suppressVariationsInInjectedValues(String content) {
     private static String replaceUnpredictableValuesWithPlaceholders(String content) {
         Date now = new Date();
         SimpleDateFormat sdf = new SimpleDateFormat("EEE, dd MMM yyyy, ");
+        //A page can have only one time zone as we cannot have session with two different time zones.
```

Comment:
You should only mention that the time zone of expected `dateTimeNow` is taken from the content.